server/src/main/scala/net/wiringbits/apis/EmailApi.scala

```diff
@@ -0,0 +1,51 @@
+package net.wiringbits.apis
+
+import com.amazonaws.regions.Regions
+import com.amazonaws.services.simpleemail.AmazonSimpleEmailServiceClientBuilder
+import com.amazonaws.services.simpleemail.model.*
+import net.wiringbits.apis.models.EmailRequest
+import net.wiringbits.config.EmailConfig
+
+import javax.inject.Inject
+
+class EmailApi @Inject() (
+    emailConfig: EmailConfig
+) {
+
+  /* To send mails to unverified accounts you need to configure aws to production mode.
```

Comment:
You need to update the readme to mention how to configure this, this comment is not in the right place